### ISSUE TITLE
Segmentation fault (core dumped)................] ETA: 0ms | Step: 0ms

### ISSUE THREAD

**Issue by atefeh-f** [atefeh-f/hello#1]
hi,
my system have core i5 and 4 GB RAM. i running a file of lua on torch library. my code is about deep learning.
please help me. my error is:

==> processing options
==> executing all
data/toy//train.csv
Gene Count: 10
Number of entries: 1000
Number of features: 5
Read entries!!
data/toy//valid.csv
Gene Count: 10
Number of entries: 1000
Number of features: 5
Read entries!!
==>DeepChrome Classification Model
==> define parameters
==> construct model
==> here is the model:
nn.Sequential {
[input -> (1) -> (2) -> (3) -> (4) -> (5) -> (6) -> (7) -> (8) -> (9) -> (10) -> output]
(1): nn.TemporalConvolution
(2): nn.ReLU
(3): nn.TemporalMaxPooling
(4): nn.View(900)
(5): nn.Dropout(0.500000)
(6): nn.Linear(900 -> 625)
(7): nn.ReLU
(8): nn.Linear(625 -> 125)
(9): nn.ReLU
(10): nn.Linear(125 -> 2)
}
==> define loss
==> here is the loss function:
nn.ClassNLLCriterion
==> loading data
==> defining some tools
==> configuring optimizer : SGD
==> defining training procedure
==> defining test procedure
==> training!
==> doing epoch on training data:
Segmentation fault (core dumped)................] ETA: 0ms | Step: 0ms
